python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -74,8 +79,20 @@ def __init__(self, config: Mapping[str, str], log):  # pylint: disable=too-many-
         self.autocommit = self.conn_args.get("autocommit", False)
         self.log = log
 
+    @public
     @contextmanager
     def get_connection(self, raw_conn=True):
+        """Gets a connection to Snowflake as a context manager.
+
+        If using the execute_query, execute_queries, or load_table_from_local_parquet methods,
+        you do not need to create a
+        connection using this context manager.
+
+        Args:
+            raw_conn: If using the sqlalchemy connector, set raw_conn to True to create a raw
+                connection. Defaults to True.
+
```

Comment:
TODO - add an example here, but need to find a fn on the connection that isn't executing a query